app/views/training_modules/_debug.html.slim

```diff
@@ -8,6 +8,7 @@
       li milestone: #{@module_progress.milestone}
       li resume page: #{@module_progress.resume_page&.name}
       li last page: #{@training_module.module_items.last.name}
```

Comment:
We mean `certificate` here don't we

Reply:
i don't think so - just whatever the last page is (which will always be a certificate) - could also add a 'last course page'